### PR TITLE
Add low intensity mixing

### PR DESCRIPTION
**Pull request by brownd1978**
low intensity mixing option

### CODE REVIEW

File: Scripts/gen_Mix.sh [hunk 1]
```diff
@@ -98,6 +99,8 @@ if [ $nbb == "one" ]; then
   echo '#include "Production/JobConfig/mixing/OneBB.fcl"' >> mix.fcl
 elif [ $nbb == "two" ]; then
   echo '#include "Production/JobConfig/mixing/TwoBB.fcl"' >> mix.fcl
+elif [ $nbb == "low" ]; then
+  echo '#include "Production/JobConfig/mixing/LowIntensity.fcl"' >> mix.fcl
 else
   echo "Must specify 'one' or 'two' booster batches"
```
Comment (kutschke) [inline Scripts/gen_Mix.sh:105]: Do you want to update the error message to include 'low' as an option?

File: JobConfig/mixing/LowIntensity.fcl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#
+# Low intensity beam configuration
+#
+# original author: Dave Brown, LBNL
+#
+physics.producers.PBISim.SDF: 0.6
+physics.producers.PBISim.extendedMean: 1.58e5 #  1% of a single booster batch
```
Comment (kutschke) [inline JobConfig/mixing/LowIntensity.fcl:7]: A small clarification on the comment.  This is the 1% of one pulse from the single booster batch configuration, not 1% of a single booster batch.  So maybe "1% of the uncut distribution from a single booster batch".

Comment (brownd1978) [reply JobConfig/mixing/LowIntensity.fcl:7]: done

File: Scripts/gen_Mix.sh (hunk 1 above)
Comment (brownd1978) [reply Scripts/gen_Mix.sh:105]: gen_Mix has major changes in another PR Id rather not adjust this here.

**Review by soleti — APPROVED** [review]

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**